templates/jobs/smoke-test-cf.groovy.j2

```diff
@@ -28,13 +28,18 @@ job('{{ job_name }}') {
   }
   publishers {
     mailer("the-multi-cloud-paas-team@digital.cabinet-office.gov.uk", false, true)
-    {% if destroy is defined and destroy %}
     downstreamParameterized {
-      trigger("ci-cf-destroy-{{ target_environment_name }}-{{ platform }}", 'SUCCESS', true) {
-        currentBuild()
-      }
+      {% if destroy is defined and destroy %}
```

Comment:
We can use the same logic `{% if downstream_parameterized_trigger_job is defined %}` instead of `{% if destroy is defined and destroy %}`.
In the case of "smoke-and-destroy", we could replace:
```
destroy: true
```
by:
```
downstream_parameterized_trigger_job: ci-cf-destroy-{{ target_environment_name }}-{{ platform }}
```